front-end/Dockerfile

```diff
@@ -3,8 +3,8 @@ FROM node:12 as build-deps
 
 ARG REACT_APP_HASURA_GRAPHQL_URL
 ARG REACT_APP_JWT_PUBLIC_KEY
-ARG REACT_APP_NETWORK="kusama"
-ARG REACT_APP_WS_PROVIDER="wss://kusama-rpc.polkadot.io"
+ARG REACT_APP_NETWORK
```

Comment:
to avoid having to have 2 dockerfiles for kusama and polkadot, these 2 are moved to the gitlab-ci.yml